react-client/src/index.jsx

```diff
@@ -79,17 +75,60 @@ class App extends React.Component {
             images: response.data[0].images,
             shippingDate: response.data[0].shippingDate,
             details: response.data[0].details,
-            prod_name: response.data[0].prod_name
+            prod_name: response.data[0].prod_name,
+            html: response.data[0].html
           })
       })
+      .then((result) => {
+        this.findAndReplaceImage();
+      })
+      .then(() => {
+        console.log('Done');
+      })
+      .catch((err) => {
+        console.error(err);
+      })
   }
 
+  findAndReplaceImage() {
+    var htmlCopy = this.state.html;
+    var i = 0;
+    var index = 0 ;
+    // while(index <= htmlCopy.length) {
+      index = htmlCopy.indexOf("src", index);
+      index += 4;
+      String.prototype.splice = function(idx, rem, str) {
+        return this.slice(0, idx) + str + this.slice(idx + Math.abs(rem));
+      };
+    
+      htmlCopy = htmlCopy.splice(index, 0, this.state.images[i]);
+      i++;
+    // }
+    this.setState({
+      html: htmlCopy
+    })
+    //console.log(index);
```

Comment:
I don't think we want to do any manipulation of this HTML once it in the database.  Your component is doing it's job of rendering the stored html.  We have to figure out our process for creating description html which will incorporate my component

Reply:
I was trying to put images dynamically instead of hard coding them

Reply:
Which works, i is just making the it a slow process.

Reply:
gotcha, I'm not sure what the best way to go is